src/modules/45-dashboards/pages/home/DashboardForm.tsx

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import {
+  Button,
+  Formik,
+  FormikForm as Form,
+  FormInput,
+  Heading,
+  Layout,
+  ModalErrorHandler,
+  ModalErrorHandlerBinding,
+  PageSpinner,
+  SelectOption
+} from '@wings-software/uicore'
```

Comment:
Totally optional, but worth reaching for `@harness/uicore` instead of `@wings-software/uicore`.
```suggestion
} from '@harness/uicore'
```